toolchain/sem_ir/ids.h

```diff
@@ -463,18 +463,20 @@ struct NameId : public IdBase<NameId> {
 
   // An ID with no value.
   static const NameId None;
-  // The name of `self`.
-  static const NameId SelfValue;
-  // The name of `Self`.
-  static const NameId SelfType;
+  // The name of `base`.
```

Comment:
Similar reordering/sorting comment

Reply:
I think these ones *are* now alphabetical (other than `None` which is special so I kept it first).